acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb

```diff
@@ -9,8 +11,10 @@
   if not puppet_version
     fail "PUPPET_LEGACY_VERSION is not set, e.g. '3.7.5'"
   end
-  install_package host, 'puppet', puppet_version
+  hosts = [host]
+  install_puppet({:version => puppet_version})
```

Comment:
@anodelman @kevpl can we update beaker's `install_puppet` method to take an optional array of hosts, and not always assume you want to install puppet on all hosts? That way we wouldn't have to do terrible things like this.